ui/DesignSystem/Component/SourceBrowser/FileView.svelte

```diff
@@ -24,7 +23,7 @@
 
 <div data-cy="file-view">
   {#if view.kind === ViewKind.Blob}
-    <Blob {rootName} {view} on:root={onSelectRoot} on:select={onSelectCommit} />
+    <Blob {view} on:root={onSelectRoot} on:select={onSelectCommit} />
```

Comment:
```suggestion
    <Blob {view} on:select={onSelectCommit} />
```

The `on:root` binding is not necessary anymore as the origin event was removed.